DefInjected/ThingDef/Apparel_Various.xml

```diff
@@ -47,18 +47,18 @@
   <Apparel_Parka.description>Kurtka z futerkiem od wewnątrz, idealna do utrzymywania ciepła nawet w najzimniejszych temperaturach.</Apparel_Parka.description>
   
   <!-- EN: plate armor -->
-  <Apparel_PlateArmor.label>TODO</Apparel_PlateArmor.label>
+  <Apparel_PlateArmor.label>zbroja płytowa</Apparel_PlateArmor.label>
   <!-- EN: Overlapping solid plates of armor covering the entire body from neck to feet. -->
-  <Apparel_PlateArmor.description>TODO</Apparel_PlateArmor.description>
+  <Apparel_PlateArmor.description>Zbroja zrobiona z nakładających się na siebie płyt pancerza osłaniających całe ciało - od szyi po stopy.</Apparel_PlateArmor.description>
   
   <!-- EN: marine armor -->
   <Apparel_PowerArmor.label>pancerz wojskowy</Apparel_PowerArmor.label>
   <!-- EN: A suit of light powered armor. Layered plasteel-weave plates are very effective at stopping attacks, with few vulnerable joint sections. Neuro-memetic assistors allow a human to wear the armor and still move easily.\n\nArmor like this is often used by imperial janissaries and rapid-incursion space marines. -->
-  <Apparel_PowerArmor.description>TODO</Apparel_PowerArmor.description>
+  <Apparel_PowerArmor.description>Lekki mocny pancerz wojskowy. Warstwowe sploty superstali są efektywne w zatrzymywaniu ataków, przy zachowaniu tylko kilku wrażliwych punktów na łączeniach.\nPancerz tego typu jest często używany przez Cesarskich Janczarów oraz jednostki GROM-u.</Apparel_PowerArmor.description>
```

Comment:
Nie przetłumaczone:
Neuro-memetic assistors allow a human to wear the armor and still move easily. 

Hmm, różnie dajesz, raz pojedynczo raz podwójnie. 
**.\n\n**Armor like this
**.\n**Pancerz tego typu 

Tam gdzie nie ma w angielskiej wersji w ogóle to dałeś podwójnie, a tam gdzie niby ma być podwójnie to jest pojedyńczo ;>
Proponuje jakoś to zmienić.

```suggestion
  <Apparel_PowerArmor.description>Lekki mocny pancerz wojskowy. Warstwowe sploty superstali są efektywne w zatrzymywaniu ataków, przy zachowaniu tylko kilku wrażliwych punktów na łączeniach.\n\nPancerz tego typu jest często używany przez Cesarskich Janczarów oraz jednostki GROM-u.</Apparel_PowerArmor.description>
```